app/ui/Card.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import classNames from 'classnames';
 
 export default class CardComponent extends React.Component {
-  render() {
-    return (
-      <div>THIS IS CARD COMPONENT</div>
-    );
-  }
+	render() {
+		const children = React.Children.map(this.props.children, child => React.cloneElement(child));
```

Comment:
Avoid one long line of code, it's less readable.

    /* Smart but hard to read */
    const children = React.Children.map(this.props.children, child => React.cloneElement(child));

    /* Less code & okay enough readability  */
    const children = React.Children.map(this.props.children, child => (
        React.cloneElement(child))
    );

    /* Standard & good Readability */
    const children = React.Children.map(this.props.children, (child) => {
        return React.cloneElement(child))
    };

    /* Readable but tedious */
    const children = React.Children.map(this.props.children, function (child) {
        return React.cloneElement(child))
    };